starter-code/src/viking.js

```diff
@@ -1,11 +1,114 @@
 // Soldier
-class Soldier {}
+class Soldier {
+  constructor(health, strength) {
+    this.health = health;
+    this.strength = strength;
+  }
+
+  attack() {
+    return this.strength;
+  }
+
+  receiveDamage(theDamage) {
+    this.health -= theDamage;
+  }
+}
+
+// var soldier = new Soldier(150, 300);
 
 // Viking
-class Viking {}
+class Viking extends Soldier {
+  constructor(name, health, strength) {
+    super(health, strength);
+    this.name = name;
+  }
+
+  receiveDamage(damage) {
+    this.health -= damage;
```

Comment:
because it's an extend class and receiveDamage() is already declare in Soldier, you can do:
receiveDamage(damage){
  super.receiveDamage(damage)
  ...add your conditions and sentences
}